handle_triggers.go

```diff
@@ -112,6 +152,59 @@ func handleTriggers(
 	return nil
 }
 
+func getTriggerItemsHistory(
+	zabbix *Zabbix,
+	triggers []Trigger,
+) (map[string]ItemHistory, error) {
+	history := map[string]ItemHistory{}
+
+	itemIDs := []string{}
+	for _, trigger := range triggers {
+		if len(trigger.Functions) > 0 {
+			itemIDs = append(itemIDs, trigger.Functions[0].ItemID)
+		}
+	}
+
+	items, err := zabbix.GetItems(Params{
+		"itemids": itemIDs,
+	})
+	if err != nil {
+		return nil, hierr.Errorf(
+			err,
+			`can't obtain items of triggers`,
+		)
+	}
+
+	err = withSpinner(
+		":: Requesting history for items of triggers",
+		func() error {
+			for _, item := range items {
+				lastValues, err := zabbix.GetHistory(Params{
+					"history": item.ValueType,
+					"itemids": item.ID,
```

Comment:
Because it will return more than one history record for every given item, but I need only latest value.